README.md

```diff
@@ -6,6 +6,8 @@
 
 `LINKRESOLVER_BASEURL`: base url for our link resolver. `https://mit.primo.exlibrisgroup.com/discovery/openurl?institution=01MIT_INST&rfr_id=info:sid/mit.tacos.api&vid=01MIT_INST:MIT` is probably the best value unless you are doing something interesting.
 
+`ORIGINS`: comma-separated list of domains allowed to connect to (and thus query or contribute to) the application. If not defined, no external connections will be permitted.
```

Comment:
Maybe add a note that non-standard ports also need to be included (i.e. `localhost` will not work when running on `localhost:5000`, etc)